### ISSUE TITLE
broken /users/:username/feature_previews

### ISSUE THREAD

**Issue by the-j0k3r** [StylishThemes/GitHub-Dark#1031]
* **Browser**:  Chrome Version 78.0.3904.97 (Official Build) (64-bit)
* **Operating System**: Win 10 pro 1809
* **Link to page with the issue**:  Click yourself and goto Features Preview

![Capture](https://user-images.githubusercontent.com/31389848/68854302-14d07900-06dc-11ea-9e5c-6597769bf604.PNG)

* **Screenshot**:
![Capture](https://user-images.githubusercontent.com/31389848/68854228-eeaad900-06db-11ea-892f-37862e4dc69d.PNG)

```html
<div class="col-6 border-right height-fit overflow-auto">
          <ul>
              <li class="d-flex flex-row">
                <!-- '"` --><!-- </textarea></xmp> --><form data-replace-remote-form="true" class="width-full" action="/users/NicklaTome/feature_previews" accept-charset="UTF-8" method="get"><input name="utf8" type="hidden" value="✓">
                  <input type="hidden" name="selected_slug" value="notifications_v2">
                  
<button class="d-inline-flex width-full p-3 Box-row--hover-blue border-bottom text-left border-0 bg-blue" type="submit">
  <div class="mr-2">
      <span class="text-white"><svg class="octicon octicon-eye-closed" viewBox="0 0 16 13" version="1.1" width="16" height="13" aria-hidden="true"><path fill-rule="evenodd" d="M14.822.854a.5.5 0 10-.707-.708l-2.11 2.11C10.89 1.483 9.565.926 8.06.926c-5.06 0-8.06 6-8.06 6s1.162 2.323 3.258 4.078l-2.064 2.065a.5.5 0 10.707.707L14.822.854zM4.86 9.403L6.292 7.97A1.999 1.999 0 016 6.925c0-1.11.89-2 2-2 .384 0 .741.106 1.045.292l1.433-1.433A3.98 3.98 0 008 2.925c-2.2 0-4 1.8-4 4 0 .938.321 1.798.859 2.478zm7.005-3.514l1.993-1.992A14.873 14.873 0 0116 6.925s-3 6-7.94 6a6.609 6.609 0 01-2.661-.57l1.565-1.566c.33.089.678.136 1.036.136 2.22 0 4-1.78 4-4 0-.358-.047-.705-.136-1.036zM9.338 8.415l.152-.151a1.996 1.996 0 01-.152.151z"></path></svg></span>
  </div>
  <div class="col-10">
    <h4 class="css-truncate css-truncate-target text-normal f5 text-gray-dark text-bold text-white" style="max-width: 100%;">Notifications Beta</h4>
      <p class="note pr-3 text-white">
        Quickly find and take action on the notifications that matter to you.
      </p>
  </div>
</button>

</form>              </li>
              <li class="d-flex flex-row">
                <!-- '"` --><!-- </textarea></xmp> --><form data-replace-remote-form="true" class="width-full" action="/users/the-j0k3r/feature_previews" accept-charset="UTF-8" method="get"><input name="utf8" type="hidden" value="✓">
                  <input type="hidden" name="selected_slug" value="file_codeowners">
                  
<button class="d-inline-flex width-full p-3 Box-row--hover-blue border-bottom text-left border-0 " type="submit">
  <div class="mr-2">
      <span class="text-gray"><svg class="octicon octicon-eye-closed" viewBox="0 0 16 13" version="1.1" width="16" height="13" aria-hidden="true"><path fill-rule="evenodd" d="M14.822.854a.5.5 0 10-.707-.708l-2.11 2.11C10.89 1.483 9.565.926 8.06.926c-5.06 0-8.06 6-8.06 6s1.162 2.323 3.258 4.078l-2.064 2.065a.5.5 0 10.707.707L14.822.854zM4.86 9.403L6.292 7.97A1.999 1.999 0 016 6.925c0-1.11.89-2 2-2 .384 0 .741.106 1.045.292l1.433-1.433A3.98 3.98 0 008 2.925c-2.2 0-4 1.8-4 4 0 .938.321 1.798.859 2.478zm7.005-3.514l1.993-1.992A14.873 14.873 0 0116 6.925s-3 6-7.94 6a6.609 6.609 0 01-2.661-.57l1.565-1.566c.33.089.678.136 1.036.136 2.22 0 4-1.78 4-4 0-.358-.047-.705-.136-1.036zM9.338 8.415l.152-.151a1.996 1.996 0 01-.152.151z"></path></svg></span>
  </div>
  <div class="col-10">
    <h4 class="css-truncate css-truncate-target text-normal f5 text-gray-dark text-bold text-gray-dark" style="max-width: 100%;">File Codeowners</h4>
      <p class="note pr-3 ">
        Show the CODEOWNERS for a single file. Use CODEOWNERS to define individuals or teams that are responsible for code in a repository.
      </p>
  </div>
</button>

</form>              </li>
          </ul>
        </div>
```
selectors maybe .`Box-row--hover-blue` I cant get a color off of that 

and
```css
.Box-row--hover-blue:hover {
    background-color: #f1f8ff;
}
```

**Comment by xt0rted** [comment]
This was fixed in fd9a93a1c5fe52b67e3abec19c462232009e868d but since I can't do releases anymore I was leaving that to someone else.

**Comment by the-j0k3r** [comment]
Install ver globally and then run the desired

```
ver -p -d patch github-dark.css github-dark.user.css
ver -p -d minor github-dark.css github-dark.user.css
ver -p -d major github-dark.css github-dark.user.css
```
